shared-module/displayio/__init__.c

```diff
@@ -100,6 +103,11 @@ void reset_displays(void) {
                         continue;
                     }
                 #endif
+                #ifdef BOARD_USE_INTERNAL_SPI
+                    if (original_spi == (mp_obj_t)(&spi)) {
```

Comment:
Please rename spi so that it's clearer what it is. The name was fine when it was private but now it's not. Maybe `supervisor_flash_spi_bus`?

Reply:
Why `supervisor`? Wouldn't `board` be more appropriate?

Reply:
I'd do supervisor because it's the supervisor that initializes it. It is only in board when something else is on the bus besides the flash.

Reply:
Oh wait I see, you mean the actual SPI bus, `&spi`.

Reply:
Yup, yup.